contracts/dapis/proxies/DapiProxy.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IDapiProxy.sol";
+
```

Comment:
Maybe add natspec comments? (on both proxy contacts)

Reply:
I seem to have forgotten

Reply:
https://github.com/api3dao/airnode-protocol-v1/issues/65